src/modules/head/head.service.ts

```
@@ -0,0 +1,28 @@
+@Injectable()
+export class HeadService{
+    constructor(
+        @InjectRepository(HeadEntity, 'mongoConnection')
+        private readonly headEntityRepo: Repository<HeadEntity>
+    ){}
+
+    async headIsRegistered(cpf: string, email: string): Promise<Boolean>{
+        const headCPF = await this.headEntityRepo.findOne({where: {cpf}});
+        const headEmail = await this.headEntityRepo.findOne({where: {email}});
+
+        return headCPF || headEmail ? true : false;
```

Biome (1.9.4)

[error] Unnecessary use of boolean literals in conditional expression.

Simplify your code by directly assigning the result without using a ternary operator.
If your goal is negation, you may use the logical NOT (!) or double NOT (!!) operator for clearer and concise code.
 Check for more details about NOT operator.
Unsafe fix: Remove the conditional expression with


(lint/complexity/noUselessTernary)